cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -87,6 +87,7 @@ angular.module(PKG.name + '.commons')
           params = {};
           angular.copy($scope.params, params);
           params.max = 50;
+          params.escape = false;
```

Comment:
No it's not. The one is for polling data, and the other is for $resource which we use after the poll of the logs have stopped. Currently we poll for data until there is a fetch for next set of logs.